src/visualization/plot_recipes.jl

```diff
@@ -58,6 +58,23 @@ struct PlotData2D{Coordinates, Data, VariableNames, Vertices} <: AbstractPlotDat
   orientation_y::Int
 end
 
+"""
+    DGMultiPlotData{Dim, uType, VariableNames}
```

Comment:
The type parameters don't match the ones below. Unless there is a good reason to expose the parameters as public API, I would prefer to handle them as internal API and omit the type parameters in the docstring.

Reply:
I'll remove the docstring - I don't think `DGMultiPlotData` should actually be public. `PlotData2D(sol)` should be all the user needs.